app/views/components/_header-notice.html.erb

```diff
@@ -25,7 +25,13 @@
     <% if list.any? %>
       <ul class="app-c-header-notice__list govuk-list govuk-list--bullet">
```

Comment:
The gem has a (recently added) component for lists now: https://components.publishing.service.gov.uk/component-guide/list

Reply:
Nice - thanks @injms.

When we get a chance, we'll try move our lists over to use the gem